Cargo.toml

```diff
@@ -13,12 +13,13 @@ homepage = "https://github.com/Unleash/unleash-types-rs"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [features]
-default = []
+default = ["wall-clock"]
+wall-clock = ["chrono/clock"]
 openapi = ["utoipa"]
 hashes = ["xxhash-rust", "base64"]
 [dependencies]
 base64 = { version = "0.22.1", optional = true }
-chrono = { version = "0.4.39", features = ["serde"] }
+chrono = { version = "0.4.39", default-features = false, features = ["serde", "std"] }
```

Comment:
"serde" + "std" + "clock" covers all the built in features outside of wasmbind, which I don't think we want anymore